### PR TITLE
Add example OIDC setup for Azure Active Directory

### PR DESCRIPTION
**Pull request by AbdelHajou**
I noticed some people having trouble setting up OIDC for popular cloud providers like GCP and Azure AD. I thought I would add some documentation on how I set up OIDC with my Azure AD tenant. Hopefully, someone will contribute documentation for GCP (#1556) and AWS too.

### CODE REVIEW

File: docs/_docs/getting-started/openidconnect-configuration.md [hunk 1]
```diff
@@ -170,3 +187,31 @@ $ curl https://auth.example.com/auth/realms/example/protocol/openid-connect/user
...
+
+1. Add an app registration for Dependency-Track to your Azure AD tenant:
+   * Name: `Dependency-Track`
+   * Supported account types: `Accounts in this organizational directory only`
+   * Redirect URI (optional): Leave empty for now
+
+
+2. Under Authentication, add two Redirect URIs for Single Page Applications and leave all settings at default:
+   * https://dependencytrack.example.com/static/oidc-callback.html
+   * https://dependencytrack.example.com/static/oidc-callback.html?redirect=%2fdashboard
```
Comment (nscuro) [inline docs/_docs/getting-started/openidconnect-configuration.md:207]: This redirect URL is no longer necessary as of v4.4.0. The redirect target (e.g. `/dashboard`) is now stored somewhere else and is not part of the URL anymore. 

```suggestion
```

File: docs/_docs/getting-started/openidconnect-configuration.md [hunk 2]
```diff
@@ -170,3 +187,31 @@ $ curl https://auth.example.com/auth/realms/example/protocol/openid-connect/user
...
 > Dependency-Track associates every OpenID Connect user with their subject identifier (`sub` claim of the access token) upon first login.
 > If a user with the same name but a different subject identifier attempts to log in via OIDC, Dependency-Track will refuse to authenticate that user. This is done to prevent account takeovers, as some identity providers allow users to change their usernames. Also, uniqueness of usernames is not always guaranteed, while the uniqueness of subject identifiers is.
+
+### Azure Active Directory app registration
+
+The following steps demonstrate how to setup OpenID Connect with Azure Active Directory.
+
+> This guide assumes that:
+>   * the Dependency-Track frontend has been deployed to `https://dependencytrack.example.com`
+>   * an Azure Active Directory tenant has been created with name `dt-example`
```
Comment (nscuro) [inline docs/_docs/getting-started/openidconnect-configuration.md:197]: Is the name `dt-example` important? It isn't mentioned anywhere in the following steps.

```suggestion
>   * an Azure Active Directory tenant has been created
```

**Review by nscuro — CHANGES_REQUESTED** [review]
Thanks for providing a PR this quickly, @AbdelHajou! Two minor suggestions for changes, otherwise it LGTM.

**Comment by AbdelHajou** [comment]
Thanks for the review @nscuro , I've resolved your feedback

**Review by nscuro — APPROVED** [review]

**Pull request merged**